### PR TITLE
Add connection to server through IPv6 interface

### PR DESCRIPTION
**Pull request by tuckyapps**
Adds option to establish connection with openvpn server through IPv6. This way, if your ISP is already giving you an IPv6 address, you will still be able to connect to your server.
The IPv6 address is added on client's configuration file, below the IPv4 remote instruction. This way, a user will always try to connect to the IPv4 one, and only if it has no luck, it will then try to connect using IPv6.

Note: When using udp, client will need to wait until keepalive time elapses before it tries to connect through IPv6. If you would like this switch to be fast, I suggest you use TCP protocol instead.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -268,6 +267,6 @@ You can [say thanks](https://saythanks.io/to/Angristan) if you want!
 
 ## Credits & Licence
 
-Many thanks to the [contributors](https://github.com/Angristan/OpenVPN-install/graphs/contributors) and Nyr's original work.
+Many thanks to the [contributors](https://github.com/tuckyapps/OpenVPN-install/graphs/contributors) and Nyr's original work.
```
Comment (robertkraig) [inline README.md:270]: You should remove all references to `tuckyapps` if you're requesting a P.R. like this. You contribute to the Project, you don't change it to go to your repo.

**Review by robertkraig — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md:270]: Agreed.

**Comment by ThePigPeppa** [comment]
this is a bad idea.

**Comment by tuckyapps** [comment]
Hi!

What do you mean?

What should I do?

Thanks in advance,

Best,

Santiago Rosenblatt

**Comment by angristan** [comment]
Hello,

Can you rebase your PR against master?

**Comment by tuckyapps** [comment]
How about now?

I'm sorry, did not know my commits were still influencing the pull request.

Back in January, I was asked by my company to update the comments so they
point to my repo in case others would like to download it, increasing
security and more control over the code.

Is it okay now?

Thanks again!

**Comment by angristan** [comment]
Hello,

the diff is really messy right now, I would advise starting again from master, checking out a new branch, then adding the modifications regarding this PR then force-pushing the branch, please 🙏

**Pull request closed without merging**